app/react/components/form-components/EnvironmentVariablesFieldset/StackEnvironmentVariablesPanel.tsx

```diff
@@ -23,7 +23,7 @@ export function StackEnvironmentVariablesPanel({
         <div>
           You may use{' '}
           <a
-            href="https://docs.portainer.io/v/2.20/user/docker/stacks/add#environment-variables"
+            href="https://docs.portainer.io/v/2.22/user/docker/stacks/add#environment-variables"
```

Comment:
Can we grab and parse the version from useSystemStatus() ? Or even make a custom hook called useDocsVersion?

It seems like extra maintenance is required in future if kept as is

Reply:
ooo useDocsUrl() already exists, please use that

Reply:
Thanks @testA113 . It came in my mind when I was updating the version but didnt realize that there is a method already